data/core/units/undead/Necromancer.cfg

```diff
@@ -6,15 +6,18 @@
     race=human
     image="units/undead-necromancers/necromancer.png"
     profile="portraits/humans/necromancer.webp"
-    hitpoints=70
+    hitpoints=76
     movement_type=smallfoot
+    [resistance]
+        impact=90
```

Comment:
remind me what its impact resistance would be without this? i.e., is this a buff or a nerf?

Reply:
It is 10%. So it is a buff.